meson.build

```diff
@@ -259,19 +269,29 @@ configure_file(input : 'src/config.h.in', output : 'config.h',
 
 # tests
 #
-# We use dictionaries for test declarations, dictionaries were added
-# in meson 0.47.0.
-#
-if meson.version().version_compare('>= 0.47.0')
-  # Some tests use relative paths; in meson 0.56.0 this can be replaced
-  # with meson.project_source_root().
-  project_source_root = meson.current_source_dir()
-  subdir('tests')
-endif
+# Some tests use relative paths; in meson 0.56.0 this can be replaced
+# with meson.project_source_root().
+project_source_root = meson.current_source_dir()
+subdir('tests')
 
 
 # dosbox executable
 #
 executable('dosbox', ['src/main.cpp', 'src/dosbox.cpp'],
            dependencies : [threads_dep, sdl2_dep] + internal_deps,
-           include_directories : incdir)
+           include_directories : incdir,
+           install : true)
+
+# additional files for installation
+#
+install_man('docs/dosbox.1')
+
+data_dir     = get_option('datadir')
+licenses_dir = data_dir / 'licenses' / 'dosbox-staging'
+doc_dir      = data_dir / 'doc' / 'dosbox-staging'
```

Comment:
These new paths are nice and easy to read!

Reply:
Yup, it's quite nice :) Importantly, we don't need to worry about path separators in user-supplied strings :)